pkg/common/provisioner_utils.go

```diff
@@ -1,39 +1,65 @@
-package lvset
+package common
 
 import (
 	"context"
 	"fmt"
+	"hash/fnv"
 	"path/filepath"
 
 	"github.com/go-logr/logr"
-	localv1alpha1 "github.com/openshift/local-storage-operator/pkg/apis/local/v1alpha1"
-	"github.com/openshift/local-storage-operator/pkg/common"
 	corev1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
 	provCommon "sigs.k8s.io/sig-storage-local-static-provisioner/pkg/common"
+	provDeleter "sigs.k8s.io/sig-storage-local-static-provisioner/pkg/deleter"
 )
 
-func (r *ReconcileLocalVolumeSet) createPV(
-	obj *localv1alpha1.LocalVolumeSet,
+// GenerateMountMap is used to get a set of mountpoints that can be quickly looked up
+func GenerateMountMap(runtimeConfig *provCommon.RuntimeConfig) (sets.String, error) {
+	type empty struct{}
```

Comment:
Not sure I didn't miss something: is this empty struct type used somewhere?

Reply:
Good catch, I was going to implement a string set and then I realized we already have a library for it.

Reply:
I'll fix it in a follow-up